index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>UI Challenge 2</title>
+    <link rel="stylesheet" href="style/reset.css">
+    <link rel="stylesheet" href="style/base.css">
+    <link rel="stylesheet" href="style/layout.css">
+    <link rel="stylesheet" href="style/modules.css">
+    <link rel="stylesheet" href="style/normalize.css">
+</head>
+<body>
+  <header>
+    <h1>UI Challenge - Custom Form Elements</h1>
+  </header>
+
+  <section class="main-content">
+    
+    <form>
+      <div class="text-input">
+        <div class="text1">
+          <label for="text-input">text input</label>
+          <input type="text" id="title" name="title" placeholder="placeholder text"/>
+        </div>
+
+        <div class="text2">
+          <label for="text-input-validation-err">text input w&sol; validation error</label>
+          <input type="text" id="text-input-validation-err" placeholder="placeholder text"/>
+        </div>
+      </div>
+
+      <div class="select-box">
+        <div class="selectbox1">
+          <label for="select-box">select box </label>
+            <select id="select-box">
+              <input type="text" id="select-box" placeholder="please select an option..."/>
+            </select>
+        </div>
+
+        <div class="selectbox2">
+          <label for="select-box-open">select box - open</label>
+            <select id="select-box-open">
+              <option value="option1">option 1</option>
+              <option value="option2">option 2</option>
+              <option value="option1">option 3</option>
+              <option value="option1">option 4</option>
+              <option value="option1">option 5</option>
+            </select>
+        </div>  
+      </div>
+
+      <div class="checkbox">
+        <label for="published">checkbox </label>
+        <input type="checkbox" class="check-box1">
+        <input type="checkbox" class="check-box2">
+      </div>  
+      
+      <div class="radio-buttons">
+        <label for="radio">radio </label>
+        <input type="radio" id="radio-button1">

```

Comment:
If you give these two radio buttons the same `name`, it'll ensure that when one is picked, the other is un-picked. That's how you group radio buttons together.